pinot-plugins/pinot-input-format/pinot-clp-log/src/main/java/org/apache/pinot/plugin/inputformat/clplog/CLPLogMessageDecoder.java

```diff
@@ -107,9 +106,4 @@ public GenericRow decode(byte[] payload, GenericRow destination) {
       return null;
     }
   }
-
```

Comment:
Same as below: decoder could pass offset and limit to ByteArrayInputStream instead of copying.